docs/connectionIssues.md

```diff
@@ -0,0 +1,36 @@
+Troubleshooting connection issues
```

Comment:
If you have an error attempting to connect to a server, an error message will have a 'need help' link which links to this doc. It's very much a work in progress and can definitely be cleaned up and beefed up.